src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1513,6 +1504,88 @@ public static void InvalidJsonWithCommentsSingleSegment(string jsonString, int e
             }
         }
 
+        [Fact]
+        public static void EmptyJsonIsInvalid()
+        {
+            var dataUtf8 = ReadOnlySpan<byte>.Empty;
+            var json = new Utf8JsonReader(dataUtf8, isFinalBlock: true, state: default);
+
+            try
+            {
+                while (json.Read())
+                    ;
+                Assert.True(false, "Expected JsonReaderException was not thrown with single-segment data.");
+            }
+            catch (JsonReaderException ex)
+            {
+                Assert.Equal(0, ex.LineNumber);
+                Assert.Equal(0, ex.BytePositionInLine);
+            }
+        }
+
+        [Fact]
+        public static void EmptyJsonWithinSequenceIsInvalid()
```

Comment:
Can you please move this test with the rest of the ReadOnlySequence tests, here: https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

This way, you don't need the `using System.Buffers;` here